### ISSUE TITLE
Fine-tuning from last layer

### ISSUE THREAD

**Issue by csetanmayjain** [speechbrain/speechbrain#1158]
Hi,
I'm using wav2vec2 model, and want to finetune on my own data using Libripeech transformer based recipe by changing the last layers only.
How can I finetune the weights & labels of last n layers?

Thanks

**Comment by mravanelli** [comment]
Hi,
I think here you would like to freeze the wav2vec2 model and only train the parameters of some additional layers placed on top of that, right? In this is the case, you can use the freeze parameter that you find in https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/lobes/models/fairseq_wav2vec.py#L78.

**Comment by csetanmayjain** [comment]
I just need to set `freeze=True`, and the rest will be taken care of by it? It will be like fine-tuning the wav2vec2 model, on my custom data.

**Comment by TParcollet** [comment]
Yes, this is exactly what will happen.
